dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/harvest/client/HarvestingClientServiceBean.java

```diff
@@ -44,6 +42,8 @@
     IndexServiceBean indexService;
     @EJB
     DataverseTimerServiceBean dataverseTimerService;
+    @Inject
+    private HarvestingClientsService harvestingClientsService;
```

Comment:
Is it needed here?

Reply:
Yeah, forgot to remove it after some internal tests.